### ISSUE TITLE
Small blank reminder tokens 

### ISSUE THREAD

**Issue by rebecca13578** [ThePandemoniumInstitute/botc-release#179]
### Description

At the start of night 1, small blank reminder tokens appeared next to all players. I could click on them to add text (in which case a new blank reminder token appeared further in). The text was not properly formatting to fit the reminder token, and instead extending in one long line. Where I put two reminder tokens on the player at the 3-o-clock position, the text overlapped, so as to be unreadable. I couldn't get the blank reminder tokens to disappear, even with refreshing my browser. 
![Screenshot 2024-05-15 204048](https://github.com/ThePandemoniumInstitute/botc-release/assets/170032270/8098a95f-c68a-4b85-b3a4-3bf8d0e94d13)

### Steps to reproduce

I don't know how or why they appeared. 

### Browser

Chrome

### Operating System

Windows

### Date and Time

May 15, 8:30pm PDT

**Comment by bra1n** [comment]
Hi Rebecca, am I correct in assuming that your device had a touch screen? Something like the Surface laptop? Because what you're seeing there is the interface for browsers that have touch support but no mouse, hence the always visible reminder tokens. (since you can't "mouse over" on them with a touch screen)

**Comment by rebecca13578** [comment]
Hi Steffen,Nice thinking, and sneaky of my laptop! I have an HP Spectre (which has both a touch screen and a mouse pad). Before night 1 I flipped it upside down into touch screen only mode for a few seconds. When I flipped it back upright (into normal mouse mode), the token circles stuck around for the rest of the game. Perhaps they should disappear when the mouse comes back in action, but likely this isn't a huge problem. Thanks,RebeccaOn May 17, 2024, at 1:59 AM, Steffen ***@***.***> wrote:﻿
Assigned #179 to @rebecca13578.
